configs/vax/IL.yaml

```diff
@@ -1,9 +1,9 @@
 state: IL
 
 links:
+- name: vaccine
```

Comment:
Curious about this one - the order these are in shouldn't change the outcome or the filenames, is this just to organize a bit better?